istioctl/cmd/add-to-mesh.go

```diff
@@ -119,8 +119,9 @@ func deploymentMeshifyCmd() *cobra.Command {
 	var opts clioptions.ControlPlaneOptions
 
 	cmd := &cobra.Command{
-		Use:   "deployment <deployment>",
-		Short: "Add deployment to Istio service mesh",
+		Use:     "deployment <deploy>",
```

Comment:
Why change what is in the angle brackets?  I think you mean "deploy <deployment>"